src/components/BuildBaronAndAnnotations/BuildBaronContent.tsx

```diff
@@ -68,11 +68,13 @@ const BuildBaronCore: React.FC<BuildBaronCoreProps> = ({
             removeBanner={dispatchBanner.removeBanner}
           />
 
-          <CreatedTickets
-            taskId={taskId}
-            setCreatedTicketsCount={setCreatedTicketsCount}
-            createdTicketsCount={createdTicketsCount}
-          />
+          {bbData.buildBaronConfigured && (
```

Comment:
I don't know if we should be hiding this here because the component already checks if `length > 0`, and this will eventually be used outside of the build baron.

Reply:
@malikchaya2 I addressed this in the other thread; I disagree that a tooltip is the right move, since this is currently not a feature that's available for projects that aren't configured. When it _is_ a task annotation feature, we can just make it visible at that point. Lara has agreed that this design is good, and in slack you said either a tooltip or hiding it was fine, so I'd like to move forward with this.

Reply:
Discussed over slack- moving the check into the BBCreatedTickets component to only hide the button and title and not the list.